MSAL/src/http/MSALHttpRequest.m

```diff
@@ -176,7 +177,9 @@ - (void)send:(MSALHttpRequestCallback)completionHandler
     [event setHttpMethod:request.HTTPMethod];
     [event setHttpURL:newURL];
     
-    LOG_INFO(_context, @"HTTP request %@", [MSALAuthority isKnownHost:request.URL] ? request.URL.absoluteString : [request.URL.absoluteString msalShortSHA256Hex]);
+    LOG_INFO(_context, @"HTTP request %@ - additional parameters are hidden to protect any personal identifiable information",
```

Comment:
nit: I would remove "additional parameters are hidden to protect any personal identifiable information"
Otherwise we need to follow this rule everywhere else as well where we hide PII... :)